pkg/core/bootstrap/bootstrap.go

```diff
@@ -69,8 +70,10 @@ func createDefaultMesh(runtime core_runtime.Runtime) error {
 
 	if err := resManager.Get(context.Background(), &defaultMesh, core_store.GetBy(key)); err != nil {
 		if core_store.IsResourceNotFound(err) {
-			core.Log.Info("Creating default mesh from the settings", "mesh", cfg.Defaults.Mesh)
-			defaultMesh.Spec = cfg.Defaults.Mesh
+			if err := util_proto.FromYAML([]byte(cfg.Defaults.Mesh), &defaultMesh.Spec); err != nil {
```

Comment:
Can we refactor `util_proto.FromYAML([]byte(cfg.Defaults.Mesh)` into a factory function ? And put it into respective `config` package